package.json

```diff
@@ -142,16 +142,16 @@
     "eslint-plugin-mocha": "^6.3.0",
     "eslint-plugin-node": "^10.0.0",
     "eslint-plugin-prettier": "^3.1.2",
-    "fixturify": "^1.2.0",
-    "fixturify-project": "^1.10.0",
-    "mocha": "^6.2.1",
-    "nock": "^11.7.0",
-    "nyc": "^14.1.0",
+    "fixturify": "^2.1.0",
+    "fixturify-project": "^2.1.0",
+    "mocha": "^7.1.0",
+    "nock": "^12.0.2",
+    "nyc": "^15.0.0",
```

Comment:
Changes listed here: https://github.com/istanbuljs/nyc/blob/master/CHANGELOG.md#1500-2019-12-20

Should be fine.